llvm/test/CodeGen/AMDGPU/amdgpu-split-kernel-args.ll

```diff
@@ -0,0 +1,120 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-attributes --check-globals all --version 5
+; RUN: opt -S -mtriple=amdgcn-amd-amdhsa -passes=amdgpu-split-kernel-arguments -amdgpu-enable-split-kernel-args -verify-machineinstrs < %s | FileCheck %s
```

Comment:
Missing checks of the IR pass output

Reply:
Can you elaborate? I have FileCheck for opt output and autogenerated check lines.